neural_compressor/compression/callbacks.py

```diff
@@ -200,7 +201,7 @@ def model(self, user_model):
             user_model: user are supported to set model from original framework model format
                         (eg, tensorflow frozen_pb or path to a saved model),
                         but not recommended. Best practice is to set from a initialized
-                        neural_compressor.experimental.common.Model.
+                        neural_compressor.model.Model.
```

Comment:
neual_compressor.model.Model is quite weird, it's better to use neural_compressor.Model